### ISSUE TITLE
Original filename lost after CSS filters are run

### ISSUE THREAD

**Issue by ukch** [django-compressor/django-compressor#775]
Say we have a SASS file `base.scss`, which we include as follows (using `COMPRESS_PRECOMPILERS`):

``` django
{% compress css %}
<link rel="stylesheet" type="text/scss" href="{% static "/path/to/base.scss" %}">
{% endcompress %}
```

This gets turned into the following HTML:

``` html
<link rel="stylesheet" href="/static/CACHE/css/base.623d072c400f.css" type="text/css" />
```

(note the name `base.623d072c400f.css` still contains the original name 'base')

However, if we then define some filters in `COMPRESS_CSS_FILTERS` (for example, [django-compressor-autoprefixer](https://pypi.python.org/pypi/django-compressor-autoprefixer/0.0.1)), the generated HTML changes to this:

``` html
<link rel="stylesheet" href="/static/CACHE/css/048fe983d13b.css" type="text/css" />
```

(note that the name `048fe983d13b.css` no longer contains the word 'base')

Is there a way to retain the original filename in this situation?

**Comment by karyon** [comment]
from a quick glance it seems like this behaviour depends on `COMPRESS_ENABLED` being True or not, not whether any filters are defined, although i'm not sure. do you by any chance remember whether you changed `COMPRESS_ENABLED` as well?

**Comment by ukch** [comment]
@karyon sorry I don't remember, although that rings a bell and might certainly be the case. Either way, it would still be great to remember filenames in this case.

**Comment by karyon** [comment]
i think this is mostly fixed or at least made obsolete by #861
